### DISCUSSION TITLE
MappingFunction for Multiple Sensors

### DISCUSSION THREAD

**Discussion by mattibatti** [tttapa/Control-Surface#1012] [General]
First of all, thank you so much for doing this - this library is amazing.

I am making a controller with multiple sensors and I am wondering - do I need to define an individual MappingFunction for each controller? Or is there a way to pass parameters to the MappingFunction and use it flexibly for the different sensors?

Thanks!!

**Comment by tttapa** [comment]
For future reference: #483
